lmdeploy/turbomind/utils.py

```diff
@@ -121,6 +121,20 @@ def get_gen_param(cap,
     return gen_param
 
 
+def get_model_name_from_workspace_model(model_dir: str):
+    """Get model name from workspace model."""
+    from configparser import ConfigParser
+    triton_model_path = os.path.join(model_dir, 'triton_models')
+    if not os.path.exists(triton_model_path):
+        return None
+    ini_path = os.path.join(triton_model_path, 'weights', 'config.ini')
```

Comment:
May we add path check here
```python3
            if not os.path.exists(ini_path):
                return None
```

Reply:
I think the path `os.path.join(model_dir, 'triton_models', 'weights')` should be enough to distinguish between two model sources. And we don't check the ini file because it will throw `FileNotFoundError` if the file doesn't exist,

Reply:
> I think the path `os.path.join(model_dir, 'triton_models', 'weights')` should be enough to distinguish between two model sources. And we don't check the ini file because it will throw `FileNotFoundError` if the file doesn't exist,

make sense